components/Tabs/src/MDCTabBar.h

```diff
@@ -84,6 +84,31 @@ IB_DESIGNABLE
 /** Ink color for taps on tab bar items. Default: Semi-transparent white. */
 @property(nonatomic, nonnull) UIColor *inkColor UI_APPEARANCE_SELECTOR;
 
+/**
+ Display font used for selected item titles.
+ Default is the regular font from the font loader.
+ */
+@property(nonatomic, strong, null_resettable) UIFont *selectedItemTitleFont UI_APPEARANCE_SELECTOR;
+
+/**
+ Display font used for unselected item titles.
+ Default is the regular font from the font loader.
+ */
+@property(nonatomic, strong, null_resettable) UIFont *unselectedItemTitleFont
+    UI_APPEARANCE_SELECTOR;
+
+/**
+ Padding on the left and right of the tab bar's content for scrollable tabs.
+ Default depends on the horizontal size class.
+ */
+@property(nonatomic) IBInspectable CGFloat horizontalPadding UI_APPEARANCE_SELECTOR;
```

Comment:
IBInspectable and UI_APPEARANCE_SELECTOR don't always play nicely together.  (Beach ball of death in Interface Builder)  We typically prefer UI_APPEARANCE_SELECTOR if both would be applicable.  These issues may be fixed in more recent versions of Xcode.

Reply:
Here and below.

Reply:
Done